app/api/event_invoices.py

```diff
@@ -71,6 +74,21 @@ class Meta:
                                   type_='discount-code')
 
 
+class EventInvoiceListPost(ResourceList):
+    """
+    List and create Tracks
+    """
+    def before_post(self, args, kwargs, data):
+        require_relationship(['event'], data)
```

Comment:
@poush @SaptakS @shubham-padia 
I also think that user should be required. Plus updated GET permissions to `organizer` from [dev-handbook](https://github.com/fossasia/open-event/blob/master/dev-handbook/resources-permissions.md#event-invoices---eg-v1event-invoice). Please check.